packages/subgraph/src/mappings/superToken.ts

```diff
@@ -317,6 +326,11 @@ function _createAgreementLiquidatedV2EventEntity(
         event.params.rewardAmountReceiver,
     ]);
 
+    const streamRevisionId =
+        event.params.id.toHex() + "-" + event.address.toHexString();
```

Comment:
The ID of the events is guaranteed to be: `keccak256(abi.encode(sender,receiver))`?

Reply:
yeah, the `id` field on the liquidation events is always going to be the `agreementId` and liquidations only exist for flows **currently** and will be so